docs/guides/syntax-notes.md

```diff
@@ -20,7 +20,7 @@ class Cat {
 }
 
 class Cat {
-  @prop({ type: Food }) // ERROR: Use before definition
+  @prop({ type: Food }) // ERROR: Used before definition
```

Comment:
this was actually intended, because that is from what i recall the error says

Reply:
Ok. It just didn't "sound" correct. 

Scott